scripts/make_aliyun_public

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3 -u
+
+# This is a utility script to mark all the Aliyun images that have been
+# replicated across regions as publicly available images. It assumes that
+# meta.json has been updated with replicated regions + image IDs. It also
+# assumes that you want to make all images in all regions public.
+
+import argparse
+import json
+import subprocess
+import sys
+
+from upload_ami import get_aliyun_endpoint
+
+ALIYUN = 'aliyun'
+cloud = ALIYUN
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--build', help="Build ID to use")
+    args = parser.parse_args()
+
+    arch = subprocess.check_output(["cosa", "basearch"], text='UTF-8').strip()
```

Comment:
But since this is a new script that depends on coreos-assembler, we could just put it in cosa right now right?  `cosa aliyun-make-public` or so?

Reply:
Yeah! That makes sense, I can move this functionality over there.